CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Version 5.0.0
+Released 2024-05-13
+
+- Update license to [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/)
+- Add ESM and CJS builds
```

Comment:
Another change here is that we don't export unknown colorbrewer palettes. We should do one of the following:

- (A) Document that only known/tagged colorbrewer palettes are exported
- (B) Check which palettes are missing, tag and add them

Reply:
Chose (B), done.